### ISSUE TITLE
Blueman and added widgets in panel crashed after upgrading to kernel 4.15.14

### ISSUE THREAD

**Issue by NeyzanRumi** [lxqt/lxqt#1472]
##### Current Behavior
After upgrading the kernel to 4.15.14 and restarting the system I found all widgets I added in the panel removed (but not colour and style personalisation) and Blueman stopped functioning. In blueman's screen no devices were showing and the "Adaptor" and "Device" options in the toolbar were greyed out.
Other personal settings, such as recent documents in LibreOffice or browser history, were not affected.
Also, after re-adding the lost widgets to the panel in 4.15.13 they remained even after rebooting into 4.15.14, but blueman again stopped functioning.

##### Possible Solution
The only solution I have is to run my system with the 4.15.13 kernel, hoping that the problem will be solved with the next update.

##### System Information
* Distribution & Version: Fedora 27
* Kernel: 4.15.14
* Qt Version: 5.9.4
* liblxqt Version: 0.11.1.7
* Package version: 2.1.0.8alpha2 (Blueman), 5.48.3 (Bluez)

**Comment by tsujan** [comment]
If I were you, I'd search for the cause of problem elsewhere. Blueman isn't controlled by LXQt.

**Comment by NeyzanRumi** [comment]
That is so, but the widgets were also affected and it happened right after the kernel upgrade. So, as I couldn't find this happening with other DE's it seems to have something to do with LXQt - the only other place I can think of to look for the cause is in the kernel itself.

**Comment by tsujan** [comment]
BTW, your report is clean and you've included the version of liblxqt. It isn't the latest version, so the other LXQt components installed on your system should be old too. If there are good RPM packages for the latest stable LXQt, I highly recommend an LXQt upgrade.

**Comment by NeyzanRumi** [comment]
You are right, of course, but I would prefer not to do that, as the developers of the LXQt spin of Fedora 27 are still working with liblxqt 0.11.
By the way, I upgraded to kernel 4.15.15 but the issue persists.

**Comment by agaida** [comment]
ok - enough prosa - please provide meaningful dumps.

**Comment by tsujan** [comment]
Has  nothing to do with LXQt.
